site/docs/tutorial/sql_to_substrait.md

```diff
@@ -0,0 +1,245 @@
+# SQL to Substrait tutorial
+
+To provide an introduction to Substrait for those already familiar with SQL,
+this tutorial walks through the basics of Substrait by translating basic SQL
+queries. We'll present the Substrait in JSON form so it is (somewhat) readable
+for newcomers.
+
+Substrait can be a type, an expression, a relation, or a plan. We'll look at them in
```

Comment:
Substrait is a spec for plans.   A plan is made up of those building blocks but I'm not sure it is valid to say "Substrait can be a type, an expression, a relation, or a plan".